tycho-surefire/tycho-surefire-plugin/src/main/java/org/eclipse/tycho/surefire/AbstractTestMojo.java

```diff
@@ -137,7 +137,8 @@
 
     /**
      * Set this parameter to suspend the test JVM waiting for a client to open a remote debug
-     * session on the specified port.
+     * session on the specified port. If further customization of JVM debug parameters is
```

Comment:
I used the US spelling of customization (differing from my issue), I'm assuming that is preferred by Eclipse projects?

Reply:
Honestly, I have no idea :-D
@mickaelistria might can tel something about...

Reply:
I don't think there is a rule that enforce the british or US variant of a word. So whatever works ;)